src/UI/CO_InfoController.java

```diff
@@ -57,13 +58,16 @@ public boolean handleInput( InputAction action )
     switch( action )
     {
       case DOWN:
+        shiftDown++;
```

Comment:
Might want to up the scroll speed. It gets cumbersome when there is a lot of text. Or maybe scroll for about a second at the slow speed and then increase it (double it or whatever).
For extra cool, it might be nice to remove the pause between the initial keypress and the OS repeat kicking in.

Reply:
Putting this off until we start routing input through the view side rather than the controller side.